src/main/java/mdteam/ait/core/blockentities/door/AbstractDoorBlockEntity.java

```diff
@@ -54,7 +54,8 @@ public void onEntityCollision(Entity entity) {
     protected abstract void teleport(Entity entity);
 
     @Override
-    public void setTardis(Tardis tardis) {
+    public void setTardis(ITardis tardis) {
+        System.out.println("Set tardis to " + tardis);
```

Comment:
Use a proper logger if this is going into production

Reply:
Yeah no this is WIP

Reply:
WOAH COMMENT FEATRURE HOW COOL!!